### ISSUE TITLE
Inconsistent outline on focused elements

### ISSUE THREAD

**Issue by cudevmaxwell** [ravendesignsystem/rds#114]
**Describe the bug**
The focus outline on masthead elements is very prominent. 
https://github.com/ravendesignsystem/rds/blob/dac31f2f86121ed3eb9768aca535ce2a75ff8ce0/src/_blocks/masthead/_masthead.scss#L76

The focus outline on all other elements on the page is not prominent enough.
https://github.com/ravendesignsystem/rds/blob/83c80637e2f1ca9bf65ecb11e54e21e3760acfaf/src/_core/scss/base/reboot/_reboot.scss#L76-L78

**To Reproduce**
1. Go to https://ravendesignsystem.github.io/rds/dev/blocks/
2. Try to use tab/keyboard navigation to visit links on the page. 

**Expected behavior**
The focus outline should be consistent for all links, buttons, and other focus-able elements. 

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Firefox
 - Version: 77.0.1

**Additional context**
I would argue that the focus outline (focus ring) should not be styled. The browser default is probably good enough, and consistency for a11y is important.
